script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1_2.txt

```diff
@@ -64,7 +64,7 @@ I was laughing too loud, I guess."
 <Page19>
 After that, the conversation ends.
 It is as though the friendly atmosphere we had experienced until just now was a lie.
-Everyone is just a stranger, and a silence peculiar to the city dominates the train.
+Everyone is just a stranger, and a silence peculiar in a city dominates the train.
```

Comment:
I think this is a regression - the silence is a particular phenomenon that occurs in cities (is peculiar to cities), "peculiar in a city" doesn't really parse as anything.

Reply:
I took it as a silence that would be peculiar in any city, not this one specifically.

Reply:
Peculiar here doesn't mean strange though - it means 'unique to' here